tests/deploy/nemo_deploy.py

```diff
@@ -602,75 +514,52 @@ def run_inference_tests(args):
 
     result_dic = {}
 
-    if args.existing_test_models:
```

Comment:
This looks cluttered in Github UI but it's just about removing `if args.existing_test_models` case